workshops/test/test_util.py

```diff
@@ -70,6 +70,47 @@ def test_serializability_of_parsed(self):
             self.fail('Dumping person_tasks to JSON unexpectedly failed!')
 
 
+    def test_no_new_column_added(self):
+        ''' Ensure more number of fields are not added by increasing the number of columns'''
+        csv = """personal,middle,family,email,event,role,phone_no
+john,a,doe,johndoe@email.com,2015-09-01-ttt-online,learner,9991239011
+jane,a,doe,janedoe@email.com,2015-09-01-ttt-online,learner,9991239021"""
+        person_tasks, _ = self.compute_from_string(csv)
+        person = person_tasks[0]
+        self.assertTrue('phone_no' not in person)
+
+    def test_number_of_columns(self):
+        ''' Ensure number of columns remains same in case of empty fields or extra fields '''
+
+        csv = """personal,middle,family,email,event,role,phone_no
+john,a,doe,johndoe@email.com,2015-09-01-ttt-online,learner,9991239011"""
+        person_tasks, _ = self.compute_from_string(csv)
+        person = person_tasks[0]
+        self.assertEqual(len(person),7)
+
+        csv = """personal,middle,family,email,role
+john,a,doe,johndoe@email.com,learner"""
+        person_tasks, _ = self.compute_from_string(csv)
+        person = person_tasks[0]
+        self.assertEqual(len(person),7)
+        self.assertEqual(person['event'],None)
+
+        csv = """personal,middle,family,email
+john,a,doe,johndoe@email.com"""
+        person_tasks, _ = self.compute_from_string(csv)
+        person = person_tasks[0]
+        self.assertEqual(len(person),7)
+        self.assertEqual(person['event'],None)
+        self.assertEqual(person['role'],None)
```

Comment:
Please follow PEP8 (https://www.python.org/dev/peps/pep-0008/) for styling your code. Specifically:

* remove excessive blank lines before class methods (lines 71-72, etc.) and before classes themselves (lines 111-113), and within method's code (lines 339-341, 371-372, etc.)
* add whitespace after commas

You'll get best results when you install a PEP8 plugin for your editor of choice. Or use `pep8` Python command line application (https://pypi.python.org/pypi/pep8/1.6.2).